### DISCUSSION TITLE
Can any one help me with cancel token implementation and how can I cancel all pending request while changing client route. PS: I am using rtk query createApi

### DISCUSSION THREAD

**Discussion by gsauarv** [reduxjs/redux#4661] [Help]
Can any one help me with cancel token implementation and how can I cancel all pending request while changing client route. PS: I am using rtk query createApi

**Comment by markerikson** [comment]
The issue tracker here on GitHub is reserved for bug reports and feature requests. For usage questions (which is what I believe this is), please use Stack Overflow or Reactiflux where there are a lot more people ready to help you out. Thanks!
